hazelcast/src/main/java/com/hazelcast/cache/impl/CacheMergeRunnable.java

```diff
@@ -42,6 +43,7 @@
 
 class CacheMergeRunnable extends AbstractMergeRunnable<ICacheRecordStore, MergingEntryHolder<Data, Data>> {
 
+    private final SerializationService serializationService;
```

Comment:
`serializationService` can be gotten  from `AbstractMergeRunnable`

Reply:
Good one!